addons/website_event_track/views/event_type_views.xml

```diff
@@ -7,11 +7,11 @@
         <field name="inherit_id" ref="website_event.event_type_view_form"/>
         <field name="arch" type="xml">
                 <xpath expr="//span[@name='website_menu']" position='after'>
-                <span>
+                <span class="me-3">
                     <label for="website_track" string="Tracks Menu Item"/>
                     <field name="website_track"/>
                 </span>
-                <span name="website_track_proposal">
+                <span name="website_track_proposal" class="me-3">
                     <label for="website_track_proposal" string="Track Proposals Menu Item"/>
                     <field name="website_track_proposal"/>
```

Comment:
Any reason we do not do those ? (on the event.template) :disguised_face: probably a ms-1 to add on label or so?

<img width="685" height="128" alt="image" src="https://github.com/user-attachments/assets/7f883b5a-2c44-4caa-a91c-69c6e93a859f" />


Also, on the community menu there is a different structure and a missing padding... Maybe we should simply add me-1 and me-3 to all label-field couples for consistency? :) we reduce the risk of such issues

<img width="638" height="43" alt="image" src="https://github.com/user-attachments/assets/d74cabf9-38ff-4143-9552-c22a78d7b168" />